lib/dpl/provider/elastic_beanstalk.rb

```diff
@@ -39,7 +39,11 @@ def push_app
         create_bucket unless bucket_exists?
 
         if options[:zip_file]
-          zip_file = File.join(Dir.pwd, options[:zip_file])
+          if File.dirname(options[:zip_file]) == "."
```

Comment:
assuming we need the expand path we could simply use
`zip_file = File.expand_path(options[:zip_file])` @BanzaiMan